### DISCUSSION TITLE
default Cruiser for certain web addresses optionally

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#16] [Q&A] [answered]
In Android 10 it is possible to delete the default app settings and set to "always ask". In Android 13 I have not found such an option with the result that Kurviger 2 is always started when I access to kurv.gr addresses.
At the moment certain webaddresses can be made optional defauft in Cruiser. When I try to make this for plus.codes, it is always automatically set back to off. The other 3 are kept.
Additionally it could be usefull to also make kurv.gr and kurviger.de as default for Cruiser (optionally with a separate on/off switch), so that one can choose which app to open (Kurviger 1, 2 or Cruiser) by default. Or is it somehow possible to set this back to "always ask" in Android 13?

**Comment by devemux86** [comment, marked as answer]
Unfortunately newer Android complicated the url associations...
If a website registers as the owner of a url, then those urls are forwarded to its app or browser.

- Kurviger 1/2 app is registered for kurviger.de and kurv.gr (I did it)
- Google Maps may have registered plus codes and more

I think [Total Commander](https://play.google.com/store/apps/details?id=com.ghisler.android.TotalCommander) app allows to set custom associations with files?

> make kurv.gr and kurviger.de as default for Cruiser

kurviger.de and kurv.gr links require an online route calculation with the website.

**Cruiser can import offline kurviger route files** (not kurviger route calculation).

**Reply by DolomitesTom** [reply]
OK understand, thanks.
